<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Análise de Mottling</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Parâmetros</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelFolder">
         <property name="text">
          <string>Pasta de imagens:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditFolder"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonBrowse">
         <property name="text">
          <string>Selecionar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxParams">
      <property name="title">
       <string>Configurações do filtro</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelKernel">
         <property name="text">
          <string>Kernel:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxKernelX">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="value">
          <number>21</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxKernelY">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="value">
          <number>21</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSigma">
         <property name="text">
          <string>Sigma:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxSigma">
         <property name="minimum">
          <double>0.1</double>
         </property>
         <property name="maximum">
          <double>50.0</double>
         </property>
         <property name="value">
          <double>5.0</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonProcess">
      <property name="text">
       <string>Processar imagens</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableResults">
      <property name="columnCount">
       <number>4</number>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderLabels">
       <stringlist>
        <string>Arquivo</string>
        <string>Média</string>
        <string>Std</string>
        <string>MI (%)</string>
       </stringlist>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPreview">
      <property name="text">
       <string>Pré-visualização da imagem</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="imagePreview">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
